b2luigi/basf2_helper/utils.py

```diff
@@ -6,10 +6,10 @@
 def get_basf2_git_hash():
     basf2_release = os.getenv("BELLE2_RELEASE")
 
-    if basf2_release == "head":
+    if basf2_release == "head" or basf2_release is None:
         basf2_release_location = os.getenv("BELLE2_LOCAL_DIR")
 
         assert basf2_release_location
```

Comment:
This assertion will fail if basf2 is not set up and this function is called, as the release will also be `None` in that case. This is a change in behavior, as prior to that the function previously `None` when basf2 is not set up, but I don't think it is a problem.